### PR TITLE
add node-12 stretch-slim

### PR DESCRIPTION
**Pull request by pklingem**
### To test
1. `make build_12-stretch-slim`
1. `docker run local/articulate-node:12-stretch-slim node -v`
1. verify the output is `v12.9.0`

### CODE REVIEW

File: 12-stretch-slim/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+FROM node:12-stretch-slim
+
+# Let's update and install the things.
+# Run apt-get quietly (-qq) and say yes to prompts (-y)
+# See best practices for apt-get in Docker at https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run
+# Note: we install Imagemagick in our base image so we can keep it updated in one place, and so we can add a strict security policy by default (see further down)
+# Note: we install python 2 because node-gyp v5 does not support python 3, & we want child images to continue to be able to build native addons
+RUN apt-get update -qq \
+    && apt-get upgrade -y \
+    && apt-get install -y build-essential imagemagick python2.7 \
```
Comment (mgreystone) [inline 12-stretch-slim/Dockerfile]: We don't want python 2 anymore. See #57

Comment (pklingem) [reply 12-stretch-slim/Dockerfile]: shall we merge that and I'll rebase and incorporate it into the the 12 image?

Comment (mgreystone) [reply 12-stretch-slim/Dockerfile]: 🤷‍♂

Comment (pklingem) [reply 12-stretch-slim/Dockerfile]: I went ahead and removed it.

**Pull request merged**